tests/InputFiles/ScalarWave/Input1DPeriodic.yaml

```diff
@@ -39,6 +39,10 @@ StepChoosers:
   - Cfl:
       SafetyFactor: 0.2
 
+ExpFilter0:
+  Alpha: 12
```

Comment:
Why are you using `12` instead of the `36` proposed by Hesthaven?

Reply:
Hesthaven says to start with 36 and experiment. Specifically, you want to filter only as much as necessary, not more.